router-tests/modules/set_scopes_test.go

```diff
@@ -0,0 +1,76 @@
+package module_test
+
+import (
+	"github.com/stretchr/testify/require"
+	"github.com/wundergraph/cosmo/router-tests/jwks"
+	"github.com/wundergraph/cosmo/router-tests/testenv"
+	setScopesModule "github.com/wundergraph/cosmo/router/cmd/custom-set-scopes/module"
```

Comment:
Router tests should not reference to an example. Please create on in the test-suite

Reply:
That's what we currently do with the existing modules test, we use the test module. Even so, I should move this test module to a separate directory for the test suite?

Reply:
Then by accident. Modules in router/cmd/ are demonstration exampls, not intended for testing.